pkg/asset/manifests/content/tectonic/cloud-creds-secret.go

```diff
@@ -0,0 +1,24 @@
+package tectonic
+
+import (
+	"text/template"
+)
+
+var (
+	// CloudCredsSecret is the constant to represent contents of corresponding yaml file
+	CloudCredsSecret = template.Must(template.New("cloud-creds-secret.yaml").Parse(`
+---
+kind: Secret
+apiVersion: v1
+metadata:
+  namespace: kube-system
+  name: cloud-creds
+data:
+{{- if .CloudCreds.AWS}}
```

Comment:
Can we add the type of the cluster (AWS, OpenStack, etc.) so that consumers know what to expect?

Reply:
and while we're at it (providing AWS coordinates), the registry operator would also like to know what s3 bucket name should be used for the registry (if you don't supply a name we'll create one, but it's likely users may want to create their own bucket).

Reply:
@dmage  working on it, yes

Reply:
@bparees, makes sense, might not be within the scope of this PR, I'll discuss with the installer team.